### DISCUSSION TITLE
Unknown Database 'sqitch'

### DISCUSSION THREAD

**Discussion by ambs** [theory/sqitch#645] [Q&A]
Hi
I am trying to follow the mysql tutorial.
It says that the control table sqitch is created during the deploy.
Nevertheless, it seems it is not created, and I get the message 'unknown database sqitch'
Am I missing any prior step (that might be missing in the tutorial as well)?

Thank you
ambs

**Comment by theory** [comment]
There have been a few reports of this; see #601. I have yet to track down the issue.
